package.json

```diff
@@ -34,7 +35,7 @@
     "regenerator-runtime": "^0.13.7",
     "sbc-common-components": "2.4.25",
     "uuid": "^8.3.1",
-    "vue": "^2.6.11",
+    "vue": "2.6.12",
```

Comment:
@cameron-freshworks @eve-git @argush3 @andrepestana-aot @vysakh-menon-aot FYI!!!

Something happened after Vue 2.6.12 that results in numerous `npm run build` errors regarding our decorators (eg, `@Component`).

I looked at the Vue release notes and didn't see any breaking changes 🤷‍♂️ 

Note that the Vue Template Compiler, below, should be the same version as Vue, or you get a build warning.

This change is not a permanent fix -- some day we will want to update our Vue version -- but that can be another ticket.